Resources/Prototypes/Entities/Structures/Windows/plasma.yml

```diff
@@ -54,6 +54,7 @@
     - Window
   components:
   - type: Sprite
+    drawdepth: WallTops
```

Comment:
instead of giving it to all the children, just give it to the parent `WindowDirectional`

Reply:
same applies to the ot her 2